src/__mocks__/MockComponent.jsx

```diff
@@ -0,0 +1,23 @@
+import useTheme from '../Theme'
+import {ThemeProvider} from '@mui/material/styles'
+import React, {createContext} from 'react'
+
+const ColorModeContext = createContext({
+  toggleColorMode: () => {},
+})
+
+/**
+ * @param {Object} children React component(s)
+ * @return {Object} React component
+ */
+export const MockComponent = ({children}) => {
```

Comment:
I'm unfamiliar with this semantics.  Isn't it @param props and it has props.children property?

Reply:
In ES6, you can destructure object properties into function parameters: among other things, it allows the scope of the function to get at the properties of the argument without the chore of individually accessing them.

Here's a better summary of my use case and other helpful use cases: https://simonsmith.io/destructuring-objects-as-function-parameters-in-es6

Reply:
Ah of course!  I'm still blind to this syntax.  Sorry, thanks for the explanation!